Source/Core/VideoCommon/Fifo.cpp

```diff
@@ -167,7 +167,8 @@ void SyncGPU(SyncGPUReason reason, bool may_move_read_ptr)
 
     // Opportunistically reset FIFOs so we don't wrap around.
     if (may_move_read_ptr && s_fifo_aux_write_ptr != s_fifo_aux_read_ptr)
-      PanicAlert("aux fifo not synced (%p, %p)", s_fifo_aux_write_ptr, s_fifo_aux_read_ptr);
+      PanicAlertFmt("Aux FIFO not synced ({}, {})", fmt::ptr(s_fifo_aux_write_ptr),
+                    fmt::ptr(s_fifo_aux_read_ptr));
```

Comment:
Curlies, since this is now two lines inside the `if`?

Reply:
Good catch